pages/sponsor/prospectus.vue

```diff
@@ -131,65 +301,266 @@
+                    specialSponsorShipTypes, i
+                ) in specialSponsorshipTypesMd"
+                :key="`prospectus_specials_${i}`"
+                class="gap-x-8 grid sponsor-packages mt-2"
             >
-                {{ item }}
-            </li>
-        </ol>
+                <div class="section-title">
+                    <p
+                        v-for="(title, j) in $t('specialSponsorship-title')"
+                        :key="`package_title_${j}`"
+                        :class="{
+                            'inline-flex items-center justify-center text-lg h-16 md:h-28 font-bold':
+                                j === 0,
+                        }"
+                        :style="fixParagaphMarginBottom()"
+                    >
+                        {{ title }}
+                    </p>
+                </div>
+                <div
+                    v-for="specialSponsorShipType in specialSponsorShipTypes"
+                    :key="`prospectus_special_${specialSponsorShipType}`"
+                    class="md:text-sm lg:text-sm"
+                >
+                    <div
+                        class="w-16 md:w-28 h-16 md:h-28 bg-contain bg-no-repeat mb-6 mx-auto"
+                        :style="
+                            getImgStyle(
+                                specialSponsorshipTypeImgs[
+                                    `special-sponsorship-${specialSponsorShipType}`
+                                ],
+                            )
+                        "
+                    ></div>
+                    <p class="text-center">
+                        {{
+                            $t(`specialSponsorship-${specialSponsorShipType}.0`)
+                        }}
+                    </p>
+                    <p class="text-center">
+                        {{
+                            $t(`specialSponsorship-${specialSponsorShipType}.1`)
+                        }}
+                    </p>
+                    <p class="text-center">
+                        {{
+                            $t(`specialSponsorship-${specialSponsorShipType}.2`)
+                        }}
+                    </p>
+                    <ul class="list-disc">
+                        <li
+                            v-for="(item, j) in $t(
+                                `specialSponsorship-${specialSponsorShipType}.3`,
+                            )"
+                            :key="`prospectus_special_${specialSponsorShipType}_item_${j}`"
+                        >
+                            {{ item }}
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <core-hr></core-hr>
+        <!-- Note -->
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('note-title') }}</p>
+            </template>
+            <template #right-col>
+                <ul class="list-disc list-inside">
```

Comment:
nit: I personally prefer removing `list-inside` and add proper `margin-left`
<details>
<summary> comparison</summary>

current:
![image](https://user-images.githubusercontent.com/24987826/126367632-7520c4d8-8494-4e60-b59c-3f06bac7f3d8.png)

modified:
![image](https://user-images.githubusercontent.com/24987826/126367719-25d27fcd-35c5-40dc-bfec-be98f365eabb.png)
</details>

```suggestion
                <ul class="list-disc ml-4">
```